apps/ejabberd/src/ejabberd_s2s_out.erl

```diff
@@ -899,13 +899,13 @@ handle_info({send_element, El}, StateName, StateData) ->
             case catch Mod:Fun(El) of
```

Comment:
Is the "relay_to_bridge" thing needed at all? It is never called from current code, is it useful in custom implementation? Or can we remove it?